spring-integration-ws/src/test/java/org/springframework/integration/ws/DefaultSoapHeaderMapperTests.java

```diff
@@ -93,20 +100,20 @@ public void testCustomSoapHeader() {
 	@Test
 	public void testRealSoapHeader() throws Exception {
 		String soap =
-			"<soapenv:Envelope xmlns:soapenv=\"http://schemas.xmlsoap.org/soap/envelope/\">"
-				+ "<soapenv:Header>"
-					+ "<auth>"
+				"<soapenv:Envelope xmlns:soapenv=\"http://schemas.xmlsoap.org/soap/envelope/\">"
+						+ "<soapenv:Header>"
+						+ "<auth>"
 						+ "<username>user</username>"
 						+ "<password>pass</password>"
-					+ "</auth>"
-					+ "<bar>BAR</bar>"
-					+ "<baz>BAZ</baz>"
-					+ "<qux>qux</qux>"
-				+ "</soapenv:Header>"
-				+ "<soapenv:Body>"
-					+ "<foo>foo</foo>"
-				+ "</soapenv:Body>"
-			+ "</soapenv:Envelope>";
+						+ "</auth>"
+						+ "<bar>BAR</bar>"
+						+ "<baz>BAZ</baz>"
+						+ "<qux>qux</qux>"
+						+ "</soapenv:Header>"
+						+ "<soapenv:Body>"
+						+ "<foo>foo</foo>"
+						+ "</soapenv:Body>"
+						+ "</soapenv:Envelope>";
```

Comment:
Ha! that wasn't intentional, really 😄 

Will take a look what IDEA can do on the matter.

Do you tell the same to to SCSt guys when they reformat imports order or anything similar with code style?